### PR TITLE
Implement Project Cloth

### PR DESCRIPTION
**Pull request by MapleNut**

### CODE REVIEW

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -106,6 +106,58 @@
 					to_chat(user, "<span class='warning'>\The [src] is used up, but there are more wounds to treat on \the [affecting.name].</span>")
 			use(used)
 
+/obj/item/stack/medical/bruise_pack/makeshift_bandage
+	name = "makeshift bandages"
+	singular_name = "makeshift bandage"
+	desc = "Some cloth to wrap around wounds"
+	icon_state = "makebandage"
+	origin_tech = list(TECH_BIO = 1)
```
Comment (orelbon) [inline code/game/objects/items/stacks/medical.dm]: Might make more sense to make this material tech.

File: code/game/objects/items/stacks/medical.dm [hunk 2]
```diff
@@ -106,6 +106,58 @@
...
+	animal_heal = 3
+
+/obj/item/stack/medical/bruise_pack/makeshift_banage/attack(mob/living/carbon/M as mob, mob/user as mob)
```
Comment (orelbon) [inline code/game/objects/items/stacks/medical.dm]: I think defining two `as mob` could cause issues. For `mob/living/carbon/M` you dont have to set it `as mob`.

**Comment by orelbon** [comment]
Maple, we don't use PNG images for sprites, it should be a .dmi.

File: code/modules/clothing/under_cloth_recipe [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	if(is_sharp(I))
+		user.visible_message("<span class='notice'>\The [user] begins cutting up \the [src] with \a [I].</span>", "<span class='notice'>You begin cutting up \the [src] with \the [I].</span>")
+		if(do_after(user, 50, src))
+			to_chat(user, "<span class='notice'>You cut \the [src] into pieces!</span>")
+			for(var/obj/item/clothing/under/H)
+                if(var/i in 2 to rand(3,5))
+				new /obj/item/stack/material/cloth(get_turf(src))
+			qdel(src)
+		return
+	..()
```
Comment (orelbon) [inline code/modules/clothing/under_cloth_recipe]: This file isnt in a .dm format, its text only.

**Comment by MalfunctioningAI** [comment]
wake me up inside

File: code/game/objects/items/stacks/medical.dm (hunk 2 above)
Comment (MapleNut) [reply code/game/objects/items/stacks/medical.dm]: If it doesn't for the bruisepack, then it won't for this. It's basically the bruisepack with healing nerfed.

File: code/game/objects/items/stacks/medical.dm (hunk 1 above)
Comment (Andromeda-K22) [reply code/game/objects/items/stacks/medical.dm]: Agreed.

File: code/game/objects/items/stacks/medical.dm [hunk 4]
```diff
@@ -106,6 +106,58 @@
...
+				if(W.bandaged)
+					continue
+				if(used == amount)
+					break
+				if(!do_mob(user, M, W.damage/3))
+					to_chat(user, "<span class='notice'>You must stand still to bandage wounds.</span>")
+					break
+
+				if (W.current_stage <= W.max_bleeding_stage)
+					user.visible_message("<span class='notice'>\The [user] bandages \a [W.desc] on [M]'s [affecting.name].</span>", \
```
Comment (Andromeda-K22) [inline code/game/objects/items/stacks/medical.dm]: You should probably change these to reflect the makeshift nature.

File: code/game/objects/items/stacks/medical.dm (hunk 1 above)
Comment (MalfunctioningAI) [reply code/game/objects/items/stacks/medical.dm]: Correct.

File: code/game/objects/items/stacks/medical.dm (hunk 4 above)
Comment (MalfunctioningAI) [reply code/game/objects/items/stacks/medical.dm]: "The [user] bandages, to their best of capability, \a [W.desc] on [M]'s [affecting.name]. It doesn't look like much, but it'll do."

File: code/game/objects/items/stacks/medical.dm (hunk 2 above)
Comment (orelbon) [reply code/game/objects/items/stacks/medical.dm]: It shouldn't cause any problems, disregard my first comment.

File: code/game/objects/items/stacks/medical.dm (hunk 1 above)
Comment (MapleNut) [reply code/game/objects/items/stacks/medical.dm]: Then why is bruispack here?

Comment (MapleNut) [reply code/game/objects/items/stacks/medical.dm]: (MATERIAL_TECH = 1) ?

**Comment by orelbon** [comment]
Talked to maple, we will open a new PR tomorrow and i will guide him through the process.

**Pull request closed without merging**